include/seqan3/alphabet/structure/all.hpp

```diff
@@ -33,7 +33,6 @@
 // ============================================================================
 
 /*!\file
```

Comment:
Not sure here if the second \ingroup should also be removed

Reply:
yes

Reply:
No, actually not. Because that's not file documentation, but documentation for (sub)Module!